static/app/components/stream/group.tsx

```diff
@@ -133,7 +133,8 @@ function BaseGroupRow({
   }, [organization, group.id, group.owners, query]);
 
   const trackClick = useCallback(() => {
-    if (query === Query.FOR_REVIEW) {
+    const forReviewQueries: string[] = [Query.FOR_REVIEW, Query.FOR_REVIEW_OLD];
+    if (forReviewQueries.includes(query || '')) {
```

Comment:
@17hogeju also, we can just remove the code related to `inbox_tab.issue_clicked`. We block that analytics event. Might as well clean it up now so we have one fewer references to `FOR_REVIEW`

Reply:
Should the whole trackClick constant be removed?

Reply:
@17hogeju actually yea we can, we block both events in Amplitude. Also make sure to remove the references to the analytics in `static/app/utils/analytics/issueAnalyticsEvents.tsx`

Reply:
Are these what we need to remove?
https://github.com/getsentry/sentry/blob/e6c7866c4f202b2d873a3424a7569f598c605efe/static/app/utils/analytics/issueAnalyticsEvents.tsx#L98-L110

Reply:
@17hogeju should be `issues_stream.issue_clicked` and `inbox_tab.issue_clicked`